Cargo.toml

```diff
@@ -193,6 +193,8 @@ fs2 = { git = "https://github.com/tabokie/fs2-rs", branch = "tikv" }
 # Remove this when a new version is release. We need to solve rust-lang/cmake-rs#143.
 cmake = { git = "https://github.com/rust-lang/cmake-rs" }
 
+sysinfo ={ git = "https://github.com/mornyx/sysinfo", branch = "0.26-fix-cpu" }
```

Comment:
Have you attempted to fix it on https://github.com/tikv/sysinfo?

Reply:
I prefer not to patch with a personal repo.

Reply:
That makes sense. I'm going to ask for `tikv/sysinfo` permissions.